src/config/staticStyles.js

```diff
@@ -371,6 +371,53 @@ export default {
   // https://tailwindcss.com/docs/background-size
   // See dynamicStyles.js
 
+  // https://tailwindcss.com/docs/gradient-color-stops
+  'bg-gradient-to-t': {
```

Comment:
I'm sure these don't belong in the static styles - move to dynamic so the config is supported:

```js
// tailwind.config.js
module.exports = {
  theme: {
    backgroundImage: {
      'gradient-to-t': 'linear-gradient(to top, var(--gradient-color-stops))',
      'gradient-to-tr': 'linear-gradient(to top right, var(--gradient-color-stops))',
      'gradient-to-r': 'linear-gradient(to right, var(--gradient-color-stops))',
      'gradient-to-br': 'linear-gradient(to bottom right, var(--gradient-color-stops))',
      'gradient-to-b': 'linear-gradient(to bottom, var(--gradient-color-stops))',
      'gradient-to-bl': 'linear-gradient(to bottom left, var(--gradient-color-stops))',
      'gradient-to-l': 'linear-gradient(to left, var(--gradient-color-stops))',
      'gradient-to-tl': 'linear-gradient(to top left, var(--gradient-color-stops))',
    },
  },
}
```